cmd/podman/mount.go

```diff
@@ -80,35 +85,38 @@ func mountCmd(c *cli.Context) error {
 		}
 	}
 
+	if err := checkAllAndLatest(c); err != nil {
+		return err
+	}
+
+	mountContainers, err := getAllOrLatestContainers(c, runtime, -1, "all")
+	if err != nil {
+		if len(mountContainers) == 0 {
+			return err
+		}
+		fmt.Println(err.Error())
+	}
+
 	formats := map[string]bool{
 		"":            true,
 		of.JSONString: true,
 	}
 
-	args := c.Args()
 	json := c.String("format") == of.JSONString
 	if !formats[c.String("format")] {
 		return errors.Errorf("%q is not a supported format", c.String("format"))
 	}
 
 	var lastError error
-	if len(args) > 0 {
-		for _, name := range args {
+	if len(mountContainers) > 0 {
```

Comment:
Can the user still pass in a container name(s)/id(s)?  Is that handled without c.Args()?

Reply:
Yes it should be handled in the helper function.